Gemfile

```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'pry'
   gem 'simplecov'
   gem 'shoulda-matchers'
+  gem 'json'
 end
```

Comment:
Noticed it was a problem that occurred in the lesson and this was the fix. Assignment states need to get gems approved.